src/pedersen.rs

```diff
@@ -1810,7 +1812,7 @@ mod tests {
 		let value = <u64>::max_value() - 1;
 		let commit = secp.commit(value, blinding.clone()).unwrap();
 
-		let bullet_proof = secp.bullet_proof(value, blinding.clone(), nonce.clone(), None, None);
+		let bullet_proof = secp.bullet_proof(value, blinding.clone(), nonce.clone(), nonce.clone(), None, None);
```

Comment:
Not related to this line.
Would you mind to add a test in this `rewind_message()` function, to validate the new `proof_info.blinding` (not a true blinding now) when using different `rewind_nonce` and `private_nonce`?

Reply:
Sure, i can add a tests that checks the rewound blinding factor is not the correct blinding factor